doc/manifest.md

```diff
@@ -36,9 +36,56 @@ testing_develop = [
 ]
 ```
 
+### Version 2
+
+Example of `ledger_app.toml` v2 manifest 
+
+```toml
+[info]
+version = 2
+
+[app]
+build_directory = "./"
+sdk = "C"
+devices = ["nanox", "nanos+", "stax", "flex"]
+
+[use_cases]
+debug = "DEBUG=1"
+test = "DEBUG=1"
+
+[unit_tests]
+directory = "./unit-tests/"
+
+[pytest.standalone]
+directory = "tests/"
+
+[pytest.swap]
+directory = "tests_swap/"
+[pytest.swap.dependencies]
+testing_with_latest = [
+    {url = "https://github.com/LedgerHQ/app-exchange", ref = "develop"},
+    {url = "https://github.com/LedgerHQ/app-ethereum", ref = "develop"},
+]
+testing_with_prod = [
+  {url = "https://github.com/LedgerHQ/app-exchange", ref = "master"},
+  {url = "https://github.com/LedgerHQ/app-ethereum", ref = "master"}
+]
+```
+
 ### Sections
 
-#### `[app]`
+#### `[info]` (version >= 2 only)
+
+This section is required. It contains version information.
```

Comment:
I don't understand, it's a version field inside the manifest that applications need to set?
I'm not really a fan, though if we want to do this then we need to specify `manifest_version` but even then I fear the devs will bump it and wonder why it doesn't work anymore